src/NWCClient.ts

```diff
@@ -1125,33 +1164,46 @@ export class NWCClient {
   }
 
   private async _checkCompatibility() {
```

Comment:
Minor: I realised that we only check the encryption/version tag from the info event and use the same while doing the request but we don't really check it in the response, but I think it should be okay